app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,20 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag '/sign_in' do
```

Comment:
You should use named routes, instead of manually typing the url. What happens if you want to change `/sign_in` to `/login`? That could result in a lot of manual changes which could be only one change if you used names routes.